src/acvp.h

```diff
@@ -425,6 +476,63 @@ typedef struct acvp_cmac_tc_t {
 
 /*
  * This struct holds data that represents a single test case
+ * for RSA testing.  This data is
+ * passed between libacvp and the crypto module.
+ */
+typedef struct acvp_rsa_keygen_tc_t {
+    ACVP_RSA_MODE mode; // "keyGen"
+
+    char *hash_alg;
+    unsigned int  tc_id;    /* Test case id */
+    char *pub_exp;
+    char *prime_test;
+
+    BIGNUM *e;
+    unsigned char *p_rand;
+    unsigned char *q_rand;
+
+    unsigned char *xp1;
+    unsigned char *xp2;
+    unsigned char *xp;
+    unsigned char *p1;
+    unsigned char *p2;
+
+    unsigned char *xq1;
+    unsigned char *xq2;
+    unsigned char *xq;
+    unsigned char *q1;
+    unsigned char *q2;
+
+    BIGNUM *n;
+    BIGNUM *d;
+    BIGNUM *p;
+    BIGNUM *q;
+
+    unsigned char *seed;
+    unsigned int seed_len;
+    unsigned int bitlen1;
+    unsigned int bitlen2;
+    unsigned int bitlen3;
+    unsigned int bitlen4;
+
+    unsigned char *prime_seed_p2;
+    unsigned char *prime_seed_q1;
+    unsigned char *prime_seed_q2;
+
+    unsigned char *prime_result; // "prime" or "composite"
+} ACVP_RSA_KEYGEN_TC;
+
+typedef struct acvp_rsa_tc_t {
+    ACVP_CIPHER cipher;
+    ACVP_RSA_MODE mode; // "keyGen"
+    int info_gen_by_server;
+    unsigned int rand_pq;
+    unsigned int mod;
+    ACVP_RSA_KEYGEN_TC *keygen_tc;
```

Comment:
I assume the plan is to add SigGen and SigVer TC structs here when they are coded ?